### PR TITLE
Add visual certificate indicator (EXPOSUREAPP-9200)

### PR DESCRIPTION
**Pull request by axelherbstreith**
Added visual indicator in form of an error to all certificate cards in the person details screen

**Recovery Certificate:**

image

**Vaccination Certificate**

image

Test Certificate
****
image

### CODE REVIEW

**Comment by marcauberer** [comment]
@axelherbstreith doesn't the icon already exist in `ic_forward`? I think you could use this instead of creating a new icon resource.

**Review by NataliaLemmerth — DISMISSED** [review]
UA approved

File: Corona-Warn-App/src/main/res/drawable/ic_arrow_right.xml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (jurajkusnier) [inline Corona-Warn-App/src/main/res/drawable/ic_arrow_right.xml:1]: isn't this the same arrow that we use in the contact journal? `ic_contact_diary_right_arrow.xml` couldn't we reuse the asset?

Comment (axelherbstreith) [reply Corona-Warn-App/src/main/res/drawable/ic_arrow_right.xml:1]: checked both proposed arrows but unfortunately none of those match the color...

Comment (marcauberer) [reply Corona-Warn-App/src/main/res/drawable/ic_arrow_right.xml:1]: Tinting?

Comment (axelherbstreith) [reply Corona-Warn-App/src/main/res/drawable/ic_arrow_right.xml]: Done -> ic_contact_diary_right_arrow.xml is now reused with alpha="0.6"

**Comment by jurajkusnier** [comment]
Looks good, just **Impfzertifikat** is a bit off.
Also, check the layout with the increased font size.
And could you try `android:hyphenationFrequency="normal"` instead of hardcoding hyphen into the text?

image

File: Corona-Warn-App/src/main/res/drawable/ic_arrow_right.xml (hunk 1 above)
Comment (marcauberer) [reply Corona-Warn-App/src/main/res/drawable/ic_arrow_right.xml]: Thanks 👍

**Review by jurajkusnier — APPROVED** [review]

**Review by mtwalli — APPROVED** [review]
LGTM

**Pull request merged**